e2e-tests/ground-helpers.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2019 The Ground Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum LoiType {
```

Comment:
Please add tsdoc explaining what these enums are used for (i.e., why we can't just use the enums defined elsewhere in the model.

Reply:
Done.